### DISCUSSION TITLE
Possible to Disable BI Respawn System?

### DISCUSSION THREAD

**Discussion by Linxuski** [Vdauphin/HeartsAndMinds#1062] [Question and answer] [answered]
Hi!

Is it possible for me to disable the BI Respawn system that's used on the 1.21 version of H&M?

The reason I wish to do this is because I currently have 3 Zeus modules set up for my and my friends and when we die and have to respawn via that system, it removes our ability to access Zeus. Only way to regain this is to rejoin the server.

**Comment by Vdauphin** [comment, marked as answer]
Hello,
Not sure why you want to disable respawn but take a look to this: https://community.bistudio.com/wiki/Arma_3:_Respawn
Cheers

**Reply by Linxuski** [reply]
Hi!

I don't want to disable respawning completely, just the BI Respawn system implemented in v1.21. It's purely because it interferes with Zeus, i.e. it not allowing a player to access it upon respawning without rejoining the server.

Is it possible for me to set it back to how it was in earlier versions, before the implementation of the new system?

I did look through the link you sent and it seems I should be changing the 'Respawn Type' to 'BASE' but I'm unable to find anything related to an option such as that through some searches of the .sqf files.

**Reply by Vdauphin** [reply]
> I don't want to disable respawning completely, just the BI Respawn system implemented in v1.21. 

BI respawn system is the same as before. We just show the BI Respawn menu

> I did look through the link you sent and it seems I should be changing the 'Respawn Type' to 'BASE' but I'm unable to find anything related to an option such as that through some searches of the .sqf files.

The BI doc say it is in description.ext

**Reply by Linxuski** [reply]
Yaay!

I was able to disable it by removing the respawn references in the description.ext and also setting the respawn to be 'Respawn on Custom Position' in the editor on the mission (just in case anybody else was curious on how to do it).

Thanks a lot for your help ❤
